test/Microsoft.Azure.WebJobs.Host.UnitTests/CompositeTraceWriterTests.cs

```diff
@@ -120,5 +126,51 @@ public void Constructor_CreatesCopyOfCollection()
             Assert.Equal(2, textWriter.ToString().Split(new string[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries).Length);
             Assert.Equal(0, t3.GetTraces().Count);
         }
+
+        [Fact]
+        public void TextWriter_IsSynchronized()
```

Comment:
This test failed 100% of the time for me before the fix. Now passes 100% of the time.